continuedev/continue
# Index on file change

## Description

Indexes files on save. Fixes an issue where Continue wasn't reindexing the file and reloading the submenu items on save.

Now you can load a code context with @Class, change the class, then load it again with @Class and the submenu item should be loaded with the latest results from the context provider.

## Checklist


## Testing

1. Open a file with Continue.
2. Add a `console.log("Test")` to the file.
3. Ask Continue how many Test logs are in the file. Should return something like 1.
4. Add `console.log("Test2")` to the file.
5. Ask Continue how many Test logs are in the file. Should return something like 2.
